league/urls.py

```diff
@@ -1,20 +1,25 @@
+import re
 from django.conf.urls import url
+from league.models import LeagueEvent
 
 from . import views
 
+# init urlpatterns
+urlpatterns = []
+
+# make the pattern for all the events
+for (event_type,_) in LeagueEvent.EVENT_TYPE_CHOICES:
```

Comment:
```suggestion
for (event_type, _) in LeagueEvent.EVENT_TYPE_CHOICES:
```

pylint is unhappy about the missing space https://github.com/climu/openstudyroom/runs/4786201742?check_suite_focus=true